src/types.ts

```diff
@@ -59,14 +59,12 @@ export type GetOperationType = "GET_VALUE" | "GET_RULE" | "GET_OWNER" | "GET_FUN
 export type OwnerPermission = "branch_owner" | "write_function" | "write_owner" | "write_rule";
 
 export type GetOptions = {
-  is_global?: boolean,
-  is_final?: boolean,
-  is_shallow?: boolean,
-  // TODO(platfowner): Remove this option once migration is completed.
-  is_raw_result_request?: boolean,
-  include_version?: boolean,
-  include_tree_info?: boolean,
-  include_proof?: boolean,
+  is_global?: boolean;
```

Comment:
Q) I am quite new in typescript. What makes this difference? I mean , and ;

Reply:
It looks like ';' is more common for types and interfaces although ',' doesn't cause compile errors.